docs/pyrona.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Pyrona - Fearless Concurrency for Python
+---
+
+# Pyrona - Fearless Concurrency for Python
+
+As part of Project Verona, we have been developing a new ownership model for Python, called Lungfish.
+This model is designed to provide a safe and efficient way to manage memory and concurrency in Python programs.
+Pyrona is our experiments to develop this approach.
+
+
+## Plan for Python
+
+### Quick prototyping in FrankenScript
```

Comment:
It would be great with a FrankenScript heap graph here.

Reply:
Great idea. I'll generate one.